daemon/lokinet.cpp

```diff
@@ -13,16 +12,38 @@
 #include <llarp/util/service_manager.hpp>
 #endif
 
-#include <csignal>
+#include <CLI/CLI.hpp>
 
+#include <csignal>
 #include <string>
 #include <iostream>
 #include <thread>
 #include <future>
-
-#include <CLI/App.hpp>
-#include <CLI/Formatter.hpp>
-#include <CLI/Config.hpp>
+#include <cstdlib>
+#include <exception>
+#include <filesystem>
+#include <initializer_list>
+#include <memory>
+#include <optional>
+#include <stdexcept>
+#include <utility>
+
+#include "CLI/Error.hpp"
+#include "CLI/Option.hpp"
+#include "CLI/impl/App_inl.hpp"
+#include "CLI/impl/Option_inl.hpp"
+#include "fmt/core.h"
```

Comment:
The includes above here (i.e. headers that aren't part of llarp) should be <...> instead of "...".

Reply:
Actually I think *most* of them should be.

The rule we decided on for this was to use `<a/b>` for a fully qualified headers -- certainly for external library headers, but also for fully-qualified llarp headers (e.g. `#include <llarp/util/logging.hpp>` rather than `#include "llarp/util/logging.hpp"`).

For includes in the same directory or a subdirectory, though, we use double-quotes, which try relative paths first (which angle includes do not).  So for instance, inside `llarp/foo.cpp` we would prefer `"#include "bar.hpp"` to `#include <llarp/bar.hpp>`, and likewise for includes from subdirectories, i.e. `#include "util/logging.hpp"` instead of `#include <llarp/util/logging.hpp>`.

Reply:
(Personally I'm fine with `#include "../whatever.hpp"` as well, e.g. so `llarp/foo/bar.hpp` could `#include "../util/logging.hpp"` rather than `#include <llarp/util/logging.hpp>` but Jeff and Tom vetoed that idea)